apps/expo/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     },
     "jsx": "react-native",
     "types": ["nativewind/types"],
-    "checkJs": false
+    "checkJs": false,
+    "moduleSuffixes": [".ios", ".android", ".native", ""]
```

Comment:
Do you have a reference link for this?

Reply:
https://www.kylemcd.com/posts/typescript-modules-suffixes-with-react-native

https://www.typescriptlang.org/tsconfig/#moduleSuffixes

Reply:
Links from @necmettindev are fine